scripts/sync-react.js

```diff
@@ -15,21 +15,25 @@ const fetch = require('node-fetch')
 // Specify a canary version:
 //   pnpm run sync-react --version 18.3.0-next-41110021f-20230301
 //
+// Sync experimental channel:
+//   pnpm run sync-react --experimental
```

Comment:
I think we should consider always keeping the stable and experimental versions in lockstep. I can't think of a good argument for why you'd want them to be separate.

Reply:
It would be really confusing if the `experimentalReact` option not only changed the release channel but also the upstream revision of React that it corresponds to

Reply:
Great call - yes agreed that they should be aligned!